office/taskopen/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        jschlatow taskopen 2.0.0 v beta
+github.tarball_from archive
+revision            0
+
+categories          office
+license             GPL-2
+maintainers         {judaew @judaew} openmaintainer
+
+description         Tool for taking notes and open urls with taskwarrior
+long_description    {*}${description}
+
+checksums           rmd160  6b9e43507cf662942604b2e419cf115a065c7e91 \
+                    sha256  2487cc132118041b9a49a4ff956caedfa6ec70d93d3cd903601de166e2d0246a \
+                    size    35371
+
+# Sync Makefile with upstream for fix build
+patchfiles          patch-fix-Makefile.diff
+
+depends_build       port:nim port:pandoc
```

Comment:
Only minor nitpick would be to make this `depends_build-append`, but looks good.